modules/coreboot

```diff
@@ -109,6 +109,15 @@ coreboot-24.12_commit_hash := 2f1e4e5e8515dd350cc9d68b48d32a5b6b02ae6a
 #Don't reuse any coreboot buildstack for now since nothing else is based on 24.12
 $(eval $(call coreboot_module,24.12,))
 
+# d16 15h fork could use the 4.15 toolchain, but d16 is alone consuming it, so
```

Comment:
This is false. 15h fork based on 4.11.  So crossgcc buildstack could be reused for librem_l1um here once more.

Also needs to change CircleCI to reflect workspace reusal.